raiden/tests/integration/test_send_queued_messages.py

```diff
@@ -197,24 +228,33 @@ def test_payment_statuses_are_restored(
         message_handler=message_handler,
         discovery=app0.raiden.discovery,
     )
-
     app0.stop()
     del app0  # from here on the app0_restart should be used
 
     app0_restart.start()
+    for identifier in range(spent_amount):
+        identifier = identifier + 1
+        mapping = app0_restart.raiden.targets_to_identifiers_to_statuses
+        status = mapping[app1_restart.raiden.address][identifier]
+        assert status.amount == 1
```

Comment:
Note: These three asserts are checking the payment status is properly restored.

Reply:
yep! Btw where is the payment_status amount used apart from the `payment_status.matches()` here?

https://github.com/raiden-network/raiden/blob/adfe245ea8258de51c909d76364b28a06c63c5f4/raiden/raiden_service.py#L799-L811

Reply:
As far as I recall it's only used in there.